### ISSUE TITLE
Inconsistent use of GARD IDs

### ISSUE THREAD

**Issue by twhetzel** [monarch-initiative/mondo#7883]
In reviewing the term MONDO:0007804 'Pallister-Hall syndrome' for another ticket, I noticed that the GARD ID exists in two different formats. One where the number has zero padding (GARD:0007305) when added as the source value for a `seeAlso` and when added as am xref for to a synonym. Then when used as a database_cross_reference it does not have zero padding (GARD:7305).

![Screenshot 2024-06-28 at 3 19 50 PM](https://github.com/monarch-initiative/mondo/assets/2167174/737f4ec3-8f6b-42e2-a981-1ff6a9ac6b1e)

![Screenshot 2024-06-28 at 3 20 16 PM](https://github.com/monarch-initiative/mondo/assets/2167174/08818655-903b-420e-b91c-2ede002b083c)

Unclear if this is a one-off or needs to be explored further than just this example.

**Comment by sabrinatoro** [comment]
@matentzn I am assigning this one to you.

**Comment by matentzn** [comment]
This is true, the GARD evidence from synonyms is from the old version, while the GARD xrefs are new. I would suggest we remove GARD evidence from synonyms until we get preferred label lists from them.. if you agree, I can make it happen
